columbo/_interaction.py

```diff
@@ -550,6 +558,7 @@ def get_answers(
     :return: Dictionary of answers.
     :raises DuplicateQuestionNameException: One of the given questions attempts to reuse a name. This includes a
         question that was used to create `answers` if given.
+    :raises ValueError: A default value was not valid and `no_user_input` is set to `True`.
```

Comment:
This function already [raises a `ValueError`](https://github.com/wayfair-incubator/columbo/blob/3d89377998ac004571c1c9b84f9b25275ee1bdee/columbo/_interaction.py#L575) (which was undocumented), so we should update this to note the two possible causes of a value error when calling this function.

Reply:
Answered this in the previous comment.